docs/092-ht_change_voice.md

```diff
@@ -1,3 +1,60 @@
 # How To Change Your Assistant's Voice
 
-**WIP**
+## Using TTS server plugin
+
+### General
+TTS server plugin is using services hosted by OVOS community:
+https://openvoiceos.github.io/community-docs/325-members/#members-hosting-services
+
+That reduces local resource consumption. Its recommended to use TTS server plugin for resource constraint hardware (eg. RPi 3)
+
+You can host TTS server by yourself - check this [blog post for details](https://blog.graywind.org/posts/piper-tts-server-script/)
+
+### Changes 
+Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.
+
+If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config
+
+```json
+{
+    "tts": {
+        "ovos-tts-plugin-server": {
+            "voice": "amy-low"
+        }
+    }
+}
+```
+
+For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin)
+
+## Using local piper plugin
+
+### General
+Local piper plugin does all Text-To-Speech translation on your local device.
+
+### Changes
+Open the file for editing.  `nano ~/.config/mycroft/mycroft.conf`.
+
+If your file is empty, or does not have a `"tts"` section, you need to create it.  Add this to your config
+
+```json
+{
+    "tts": {
+        "module": "ovos-tts-plugin-piper",
+        "ovos-tts-plugin-piper": {
+            "voice": "alba-medium"
+        }
+    }
+}
+```
+
+For avaliable voices please refer to [piper-voices section](#voices-avaliable-for-piper-plugin)
+
+> [!WARNING]  
+> After configuration change it's important to reload it
+> (an example used with systemd services `systemctl --user restart ovos`)
```

Comment:
This is only applicable for RaspOVOS and ovos-installer (I think). It may be worth noting that other systems may have a different way to restart services.

Reply:
Ok. Just found that buildroot is also using systemd
https://openvoiceos.github.io/community-docs/050-starting_intro/
So I guess all available editions have systemd.

Reply:
I added

> If your system is not using systemd - please refer to relevant documentation

Let me know if you are happy with that

Reply:
Do you have a link to the "relevant documentation"
https://openvoiceos.github.io/community-docs/051-starting_modules/